cosmetics (#592)
cap-js/docs: node.js/core-services.md

## Implementing Services

> By default `cds.serve` creates an instance of `cds.ApplicationService` for each service definition it finds. Each instance provides generic implementations for all CRUD operations, including full support for deep document structures, declarative input validation and many other out-of-the-box features. Yet, you'd likely need to provide domain-specific custom logic, especially for custom actions and functions, or for custom validations. In the next sections, you can learn:

Reviewer: ```suggestion
By default `cds.serve` creates an instance of `cds.ApplicationService` for each service definition it finds. Each instance provides generic implementations for all CRUD operations, including full support for deep document structures, declarative input validation and many other out-of-the-box features. Yet, you'd likely need to provide domain-specific custom logic, especially for custom actions and functions, or for custom validations. In the next sections, you can learn the following:
```

- **How** to provide custom implementations?

> - **Where**, i.e. in which files, to add the implementation?

Reviewer: We try to avoid latin abbreviations. But if you can phrase it differently that would also be fine :)
```suggestion
- **Where**, that is, in which files, to add the implementation?
```
